src/pages/dashboard/AdminMenu.tsx

```diff
@@ -21,17 +22,23 @@ const StyledButton = styled(Button)`
   margin-right: 1rem;
 `;
 
-const AdminMenu: React.FC = () => {
+const AdminMenu: FC = () => {
   const user = useSelector((state: RootStore) => state.user);
+  const [showNewRegistrationButton, setShowNewRegistrationButton] = useState(false);
+  const history = useHistory();
   const { t } = useTranslation();
 
+  useEffect(() => {
+    setShowNewRegistrationButton(history.location.pathname !== '/publications/new');
+  }, [history.location.pathname]);
```

Comment:
trenger vi en egen state og useEffect for dette? virker som at det bare vil gi flere rerenders, og at vi like godt kunne sjekket mot pathname direkte

Reply:
hehehe godt poeng. fikser 👍